exchanges/kline/kline_test.go

```diff
@@ -981,3 +985,30 @@ func TestConvertToNewInterval(t *testing.T) {
 		t.Error("expected one candle")
 	}
 }
+
+func TestGetClosePriceAtTime(t *testing.T) {
+	tt := time.Now()
+	k := Item{
+		Candles: []Candle{
+			{
+				Time:  tt,
+				Close: 1337,
+			},
+			{
+				Time:  tt.Add(time.Hour),
+				Close: 1338,
+			},
+		},
+	}
+	price, err := k.GetClosePriceAtTime(tt)
+	if !errors.Is(err, nil) {
+		t.Errorf("received '%v' expectec '%v'", err, nil)
```

Comment:
```suggestion
		t.Errorf("received '%v' expected '%v'", err, nil)
```
Seems like there's a few instances of these in this file 😄